### PR TITLE
[innawood] make leather journals able to track calories

### PR DESCRIPTION
**Pull request by ghost**
#### Summary
Features "let leather journals have the ability to track calories"

#### Purpose of change
innawood-friendly way to track calories, i don't see a strong reason to not add it, keeping it in innawood pleases mainline folk and lets us have a nice way to know how much we actually consumed

#### Describe the solution
Create misc.json ( i do not know where else to put the item)  in innawood item folder, and put leather journals there with CALORIES_INTAKE flag and the option to activate to track your calories

#### Describe alternatives you've considered
none

#### Testing
Opened game, Gave myself leather journal and dog food, checked calories before and after  eating, works fine.

### CODE REVIEW

File: data/mods/innawood/items/misc.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+          "months_old_newspaper",
+          "weeks_old_newspaper",
+          "necropolis_leaflet",
+          "death_note",
+          "note_mutant_alpha_boss",
+          "evac_pamphlet"
+        ]
+      }
+    ]
+  }
```
Comment (anothersimulacrum) [inline data/mods/innawood/items/misc.json:50]: Double-check that this works.
```suggestion
    "copy-from": "leather_journal"
    "name": { "str": "leather journal" },
    "looks_like": "book_nonf_soft_psych_cggamers",
    "use_action": [ "CALORIES_INTAKE_TRACKER" ],
    "extend": { "flags": [ "TRADER_AVOID", "CALORIES_INTAKE" ] }
  }
```

Comment (Devengel) [reply data/mods/innawood/items/misc.json:50]:
```[
  {
    "copy-from": "leather_journal",
    "name": { "str": "leather journal" },
    "looks_like": "book_nonf_soft_psych_cggamers",
    "use_action": [ "CALORIES_INTAKE_TRACKER" ],
    "extend": { "flags": [ "TRADER_AVOID", "CALORIES_INTAKE" ] }
  }
]
```


this is the code i put in and it throws out this error, apologies im pretty noob
image

Comment (Devengel) [reply data/mods/innawood/items/misc.json:50]: oh im silly! ignore that

**Comment by Devengel** [comment]
apologies for the messy commit history, there was some sort of hidden character that kept misaligning a code line

**Comment by Devengel** [comment]
forgot to mention, double checked with the new code and it does work, thank you anothersimulacrum!

**Comment by AtomicFox556** [comment]
Why not add it to main game files? It seems possible to keep track of one's calories intake if you just write down everything that you eat as long as you know the amount of calories in everything you eat.

**Comment by Devengel** [comment]
@Light-Wave hey sorry for the ping, but could you let me know if this is okay and ready to be approved/merged?

**Comment by NetSysFire** [comment]
They can only approve your changes, which speeds up merging. Neither me nor them can merge them, this is restricted to the devs.

**Comment by Devengel** [comment]
ah okay thank you, thought he was part of the dev team, guess not.

**Review by Light-Wave — APPROVED** [review]

**Comment by Light-Wave** [comment]
No worries. Gave it my approval.

**Pull request merged**